src/tools/Identity/README.md

```diff
@@ -0,0 +1,140 @@
+// Place your settings in this file to overwrite the default settings
+{
+    "workbench.colorTheme": "Abyss"
+}
+
+As a prerequisite, make sure that you installed the correct PowerShell modules and versions:
+
+For Azure Stack 1904 to 1907
+
+Install the AzureRM.BootStrapper module. Select Yes when prompted to install NuGet
```

Comment:
In the identity module, Az.Accounts used. This info needs to be updated to match Az modules